packages/client/src/contexts/Web3.js

```diff
@@ -4,6 +4,8 @@ import networks from "../networks";
 import { useRouteMatch } from "react-router-dom";
 import { useFclUserBalance, useTreasury, useNFTs, useVaults } from "../hooks";
 import { CURRENT_USER_SESSION_KEY } from "constants/constants";
+import { Buffer } from "buffer";
```

Comment:
how did it work earlier without this import?

Reply:
I have no idea, probably has something to do with using babel for transpiling